.gitignore

```diff
@@ -106,3 +106,6 @@ tags
 
 # jupyter notebook checkpoints
 .ipynb_checkpoints
+
+# version file generated by rob
+procrustes/_version.py
```

Comment:
We should also replace all the **docs** with **doc** in .gitignore.

Reply:
Thanks. I will fix it in my future PR.